### DISCUSSION TITLE
Lithium + SSRIs

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#85] [Drug Interactions]
Latest research shows no interaction.

Finley, P. R. (2016). Drug Interactions with Lithium: An Update. Clinical Pharmacokinetics, 55(8), 925–941. https://doi.org/10.1007/s40262-016-0370-y

**Comment by utaninja** [comment]
Adding studies and pushing this to an issue-
Hawley CJ, Loughlin PJ, Quick SJ, Gale TM, Sivakumaran T, Hayes J, McPhee S. Efficacy, safety and tolerability of combined administration of lithium and selective serotonin reuptake inhibitors: a review of the current evidence. Hertfordshire Neuroscience Research Group. Int Clin Psychopharmacol. 2000 Jul;15(4):197-206. doi: 10.1097/00004850-200015040-00002. PMID: 10954059.

There seems to technically be a concern of SS but does not practically seem like it is a concern and is not recommending against doing this mix. I would of course, strongly caution against mixing more serotonergic drugs on top of these two.
(for example - Müller-Oerlinghausen, B. Drug Interactions with Lithium. Mol Diag Ther 11, 41–48 (1999). https://doi.org/10.2165/00023210-199911010-00004 )
or (Bauer, Michael PhD, MD;  Linden, Michael MD;  Schaaf, Berthold;  Weber, Hans J. MD. Adverse Events and Tolerability of the Combination of Fluoxetine/Lithium Compared With Fluoxetine. Journal of Clinical Psychopharmacology 16(2):p 130-134, April 1996. )

**Reply by zelixir25** [reply]
yea i usually try to find the most up to date studies esp since for lithium there's research going back decades. in general studies within the last 10-15 years and reviews and meta analyses are what i've mostly been looking for since they'll synthesize a lot of the latest research for us already and we can always go through and check *their* sources on top of it if we need more info

**Reply by utaninja** [reply]
For sure that's very fair. In general, I do like having a collection of multiple even if one study cites many.

**Comment by utaninja** [comment]
Pushed to https://github.com/TripSit/drugs/pull/125
